mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -105,6 +108,15 @@ class GaussianMixtureModel private[ml] (
   def gaussians: Array[MultivariateGaussian] = parentModel.gaussians
 
   @Since("2.0.0")
```

Comment:
I think it's better to make ```gaussians``` return ```DataFrame```, then we can get it directly at Python side. For ML, we did not need a Python wrapper like what we did at ```GaussianMixtureModelWrapper``` for MLlib. cc @jkbradley